metafields.go

```diff
@@ -65,6 +65,19 @@ func (m Metafields) GetByKey(key string) (Metafield, error) {
 	return Metafield{}, NewErrMetafieldNotFoundByKey(key)
 }
 
+// GetByNamespace gets a list of metafields under the same namespace.
+func (m Metafields) GetByNamespace(namespace string) (Metafields, error) {
+	var metafieldsInNamespace Metafields
```

Comment:
```suggestion
	metafieldsInNamespace := make(Metafields, 0, len(m))
```

Reply:
Could just call it meta fields also as it's a local variable in this function and the stored ones are m.